aioshelly/const.py

```diff
@@ -100,9 +100,9 @@
 MODEL_PRO_EM3_400 = "SPEM-003CEBEU400"
 MODEL_WALL_DISPLAY = "SAWD-0A1XX10EU1"
 # Gen3 RPC based models
-MODEL_PLUS_1_MINI_G3 = "S3SW-001X8EU"
-MODEL_PLUS_1PM_MINI_G3 = "S3SW-001P8EU"
-MODEL_PLUS_PM_MINI_G3 = "S3PM-001PCEU16"
+MODEL_1_MINI_G3 = "S3SW-001X8EU"
+MODEL_1PM_MINI_G3 = "S3SW-001P8EU"
+MODEL_PM_MINI_G3 = "S3PM-001PCEU16"
```

Comment:
These constants have not been used in the integration code yet, so I think we can rename them. If you disagree, I will restore the old names.

Reply:
I think the rename is safe

Reply:
I don't see any problem to rename them